src/app/qgsidentifyresultsdialog.cpp

```diff
@@ -78,7 +78,9 @@ QgsIdentifyResultsWebView::QgsIdentifyResultsWebView( QWidget *parent ) : QgsWeb
   setSizePolicy( QSizePolicy::MinimumExpanding, QSizePolicy::Minimum );
   page()->setNetworkAccessManager( QgsNetworkAccessManager::instance() );
   // page()->setLinkDelegationPolicy( QWebPage::DelegateAllLinks );
+#ifdef WITH_QTWEBKIT
```

Comment:
QtWebKit is no longer in the main Qt repository, so I guess it's fine to build it in a separate formula.
For reference:
https://github.com/opengisch/osgeo4travis/blob/master/docker/qt5/scripts/build-qtwebkit.sh